### DISCUSSION TITLE
GoldHEN Cheats Manager v1.1.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/PS4CheatsManager#97] [Announcements]
### Added

- Option to delete patch settings
- Support for masked pattern patches for filtering
- New Update Menu
  - Update Cheats from Web, HDD, USB (@illusion0001)
  - Update Patches from Web, HDD, USB
  - Backup local cheat files to .Zip (HDD/USB)
  - Backup local patch files to .Zip (HDD/USB)

### Misc

- Includes latest official cheat+patch pack (2023-06-01)

<hr /><em>This discussion was created from the release <a href='https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases/tag/v1.1.0'>GoldHEN Cheats Manager v1.1.0</a>.</em>

**Comment by princeduta** [comment]
by the way, isn't there a cheat for warrior orochi 4 ultimate?😢

**Comment by 20IGNTR** [comment]
hello creators  could will you take in account cheats for this games ploxx:
Dishonored 2 | CUSA03642 v01.05
Borderlands 3 | CUSA07823 v01.29
Blair Witch | CUSA18142 v01.01
DeadBolt | CUSA05931 v01.02
Hellblade Senuas Sacrifice | CUSA07527 v01.02
Assassin's Creed Syndicate | CUSA02389 v01.52
BloodBorne | CUSA09000 v01.09
COD BO3 | CUSA03004 v01.33
Injustice 2: Legendary | CUSA10992 v01.02
Watch Dogs 2 | CUSA04459 v01.00
Battlefield 1 | CUSA02429 v01.01
Star Wars Jedi: Fallen Order | CUSA12539 v01.01 
Need for Speed Heat | CUSA15081 v01.07
GhostRunner | CUSA24169 v01.13
